Batatabit/main.css

```
@@ -0,0 +1,349 @@
+td {
+  box-shadow: 2px 2px #ffa500;
+  border-radius: 4px;
+  background-color: #fdfaf7;
+  border: 1px solid #949191;
+}
+
+main .table-contain p {
+  padding: 8px;
+  background-color: #ffe9d4;
+  border-radius: 10px;
+  color: #000;
+  text-align: center;
+}
+
+main .table-contain p b {
+  font-weight: 700;
+  color: #000000;
+}
+
+main .table-contain table img {
+  width: 25px;
+  height:20px;
+  margin-left: 5px;
+}
+
+main .table-contain table .coins {
+  color: #636261;
+  font-weight: 500;
+  padding-left: 5px;
+}
+
+main .table-contain table .price {
+  color: #555353d7;
+}
+
+main .main-contain-features {
+  background-color: #201e1c;
+  color: #fff;
+  padding-bottom: 30px;
+  position: relative;
+}
+
+main .main-contain-features .batata {
+  position: absolute;
+  left: 155px;
+  width: 60px;
+  height: 45px;
+  top: calc(0% - 18.5px);
+  background-color: #282623;
+  padding: 5px;
+  border-radius: 16%;
+  box-shadow: 1px 1px #ffa500;
+}
+main .main-contain-features h2 {
+  margin: 20px;
+  padding-top: 66px;
+  font-weight: 700;
+  font-size: 27px;
+}
+
+main .main-contain-features p {
+  color: #9c9999;
+  font-size: 17px;
+}
+
+main .main-contain-features .contain-card-feature {
+  text-align: start;
+  margin: 20px;
+  background-color: #282623;
+  border-radius: 10px;
+  padding: 16px;
+  box-shadow: 1px 1px #f7931a;
+}
+
+main .contain-card-feature h2 {
+  margin: 0;
+  padding: 0;
+  font-size: 20px;
+}
+
+main .contain-card-feature .icon-feature {
+  width: 30px;
+  height: 30px;
+  margin-bottom: 10px;
+}
+
+main .main-contain-image{
+  background-image: url('assets/img/bitcoinbaby2x.jpg');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;  
+  height: 400px;
+  width: 100%;
+  margin: 0;
+}
+
+main .main-contain-image h2 {
+  margin: 0;
+  color: #fff;
+  font-size: 27px;
+  padding: 60px;
+}
+
+main .main-contain-plan {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+main .main-contain-plan h2 {
+  padding-top: 30px;
+  font-size: 24px;
+}
+
+main .main-contain-plan p {
+  font-size: 17px;
+  color: #757575;
+}
+
+main .contain-cards {
+  display: flex;
+  height: 370px;
+  overflow-x: scroll;
+  overscroll-behavior-x: contain;
+  scroll-snap-type: x proximity;
+  margin: 0px 30px;
+}
+main .contain-cards::-webkit-scrollbar {
+  background-color: #ffe9d4;
+  height: 3px;
+  width: 10px;
+}
+main .contain-cards::-webkit-scrollbar-thumb {
+  background-color: #f7931a;
+}
+
+main .main-contain-plan .outstanding {
+  padding: 10px;
+  background-color: #F7931A;
+  color: #ffe9d4;
+  border-radius: 10px;
+  position: absolute;
+  top: -15px;
+  font-weight: 700;
+}
+main .main-contain-plan .card-plan {
+  display: flex;
+  scroll-snap-align: center;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  width: 70%;
+  min-width: 280px;
+  max-width: 300px;
+  padding-top: 16px;
+  margin: 21px 0px 20px 9px;
```

```suggestion
  margin: 21px 0 20px 9px;
```

Otro detalle que no es grave, pero para que sepas que se puede simplificar la sintaxis cuando se quiere indicar 0 píxeles

---

Corregido